<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotsOfLinesAppClass</class>
 <widget class="QMainWindow" name="LotsOfLinesAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LotsOfLinesApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="centralLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="sidebarDockWidget">
   <property name="windowTitle">
    <string>Visualization Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="sidebarScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>156</width>
          <height>355</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="visualizationOptionsGroupBox">
           <property name="title">
            <string>Visualization Type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Parallel Coordinates</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">visualizationTypeButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>Colocated Paired</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">visualizationTypeButtonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dataTableDock">
   <property name="windowTitle">
    <string>Data Table</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionVisualization_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visualization Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LotsOfLinesApp.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="visualizationTypeButtonGroup"/>
 </buttongroups>
</ui>
